receiver/riakreceiver/metadata.yaml

```
@@ -0,0 +1,68 @@
+name: riakreceiver
+
+attributes:
+  riak.node.name:
+    description: The name this node uses to identify itself.
+  request:
+    description: The request operation type.
+    enum:
+      - put
+      - get
+  operation:
+    description: The operation type for index operations.
+    enum:
+      - read
+      - write
+      - delete
+
+metrics:
+  riak.node.operation.count:
+    description: The number of operations performed by the node.
+    unit: "{operation}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+    attributes: [request]
+  riak.node.operation.time.mean:
+    description: The mean time between request and response for operations performed by the node.
+    unit: us
+    gauge:
+      value_type: int
+    enabled: true
+    attributes: [request]
+  riak.node.read_repair.count:
+    description: The number of read repairs performed by the node over the last minute.
```

```suggestion
    description: The number of read repairs performed by the node.
```
Since we're using a the non-monotonic counter, this is no longer representative of the last minute.